lib/system.ps1

```diff
@@ -88,6 +88,7 @@ function Test-PathLikeEnvVar {
 function Add-Path {
     param(
         [string]$Path,
+        [string]$Scope = 'PATH',
```

Comment:
[It's confusing using it for the variable name here](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_scopes). What do you think of `$TargetEnvVar`?

Reply:
Great, I'll change it.